velox/functions/prestosql/tests/InPredicateTest.cpp

```diff
@@ -316,3 +322,25 @@ TEST_F(InPredicateTest, varbinary) {
   auto result = evaluate<SimpleVector<bool>>(predicate, input);
   assertEqualVectors(makeConstant(true, input->size()), result);
 }
+
+TEST_F(InPredicateTest, shortDecimal) {
+  auto type = DECIMAL(5, 2);
+  auto shortDecimal =
+      makeNullableShortDecimalFlatVector({10000, -10000, 20000, -20000}, type);
+  auto expected = makeFlatVector<bool>({false, true, false, true});
+
+  auto result =
+      evaluate("c0 IN (-100.00, -200.00)", makeRowVector({shortDecimal}));
```

Comment:
Similar to integers, cover cases with nulls in the list, nulls in columns, dictionary etc.

Reply:
Thanks for the suggestion, updated.